### ISSUE TITLE
Bad response error while migrate

### ISSUE THREAD

**Issue by SRLive** [joeyates/imap-backup#154]
So, I'm trying to migrate a bit mailbox and I get a Bad Response error and have to restart the migration every now and then.

I'm running the latest version.

Here is the log :
```
Traceback (most recent call last):
	29: from /usr/local/bin/imap-backup:23:in `<main>'
	28: from /usr/local/bin/imap-backup:23:in `load'
	27: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/bin/imap-backup:11:in `<top (required)>'
	26: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/lib/imap/backup/logger.rb:39:in `sanitize_stderr'
	25: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/bin/imap-backup:12:in `block in <top (required)>'
	24: from /var/lib/gems/2.7.0/gems/thor-1.2.1/lib/thor/base.rb:485:in `start'
	23: from /var/lib/gems/2.7.0/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
	22: from /var/lib/gems/2.7.0/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
	21: from /var/lib/gems/2.7.0/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
	20: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/lib/imap/backup/cli.rb:115:in `migrate'
	19: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/lib/imap/backup/cli/migrate.rb:42:in `run'
	18: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/lib/imap/backup/cli/folder_enumerator.rb:28:in `each'
	17: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/lib/imap/backup/cli/folder_enumerator.rb:28:in `glob'
	16: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/lib/imap/backup/cli/folder_enumerator.rb:28:in `glob'
	15: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/lib/imap/backup/cli/folder_enumerator.rb:32:in `block in each'
	14: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/lib/imap/backup/cli/migrate.rb:43:in `block in run'
	13: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/lib/imap/backup/migrator.rb:15:in `run'
	12: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/lib/imap/backup/account/folder.rb:43:in `create'
	11: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/lib/imap/backup/account/folder.rb:36:in `exist?'
	10: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/lib/imap/backup/account/folder.rb:150:in `examine'
	 9: from /var/lib/gems/2.7.0/gems/imap-backup-9.2.0/lib/imap/backup/client/default.rb:41:in `examine'
	 8: from /var/lib/gems/2.7.0/gems/net-imap-0.3.2/lib/net/imap.rb:579:in `examine'
	 7: from /usr/lib/ruby/2.7.0/monitor.rb:202:in `mon_synchronize'
	 6: from /usr/lib/ruby/2.7.0/monitor.rb:202:in `synchronize'
	 5: from /var/lib/gems/2.7.0/gems/net-imap-0.3.2/lib/net/imap.rb:581:in `block in examine'
	 4: from /var/lib/gems/2.7.0/gems/net-imap-0.3.2/lib/net/imap.rb:1484:in `send_command'
	 3: from /usr/lib/ruby/2.7.0/monitor.rb:202:in `mon_synchronize'
	 2: from /usr/lib/ruby/2.7.0/monitor.rb:202:in `synchronize'
	 1: from /var/lib/gems/2.7.0/gems/net-imap-0.3.2/lib/net/imap.rb:1502:in `block in send_command'
/var/lib/gems/2.7.0/gems/net-imap-0.3.2/lib/net/imap.rb:1450:in `get_tagged_response': Command Argument Error. 11 (Net::IMAP::BadResponseError)
```

**Comment by joeyates** [comment]
Hi @SRLive, thanks for opening the issue.

**Comment by joeyates** [comment]
I've released version 9.3.0 which traps errors like this - when creating the remote folder.
